src/Data/Lens/AffineTraversal.purs

```diff
@@ -0,0 +1,51 @@
+-- | This module defines functions for working with grates.
+-- |
+-- | See <http://r6research.livejournal.com/28050.html>.
```

Comment:
Should probably drop/modify this comment?

Reply:
Yes thanks!